Packs/ForcepointDLP/Integrations/ForcepointEventCollector/ForcepointEventCollector_description.md

```diff
@@ -0,0 +1,14 @@
+## Forcepoint Event Collector
```

Comment:
![image](https://github.com/demisto/content/assets/1146242/f3b6636b-0e5a-4797-b13e-36ad1ce63955)
![image](https://github.com/demisto/content/assets/1146242/ce45c067-f44b-41bc-b032-97449fd28047)
for reference on how it looks in the UI.